types/electron-store/index.d.ts

```diff
@@ -111,4 +111,4 @@ declare class ElectronStore<T = {}> implements Iterable<[string, JSONValue]> {
     [Symbol.iterator](): Iterator<[string, JSONValue]>;
 }
 
-export = ElectronStore;
+export default ElectronStore;
```

Comment:
The previous definition was correct based on the actual implementation as seen here: https://unpkg.com/electron-store@3.0.0/index.js.